languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -1398,6 +1398,12 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
 
         <rulegroup id='GENERAL_GENDER_AGREEMENT_ERRORS' name="Concordância: Geral">
             <url>https://pt.wikibooks.org/wiki/Portugu%C3%AAs/Concord%C3%A2ncia/Concord%C3%A2ncia_nominal</url>
+            <antipattern>
```

Comment:
This antipattern technically allows for:
> o que você acha dos outros regras?
> quero dar mais consistência aos outros regras

Both of which are incorrect, so I don't think we should be introducing this.

---

The key word here is the **verb**. If you have a verb that governs a certain preposition (such as 'impor', in your example), then we should allow these constructions. I'd suggest adding an extra token accounting for this verb.